core/state/managed_state_test.go

```diff
@@ -26,7 +26,7 @@ import (
 var addr = common.BytesToAddress([]byte("test"))
 
 func create() (*ManagedState, *account) {
-	db, _ := ethdb.NewMemDatabase()
+	db := ethdb.NewMemDatabase()
 	statedb, _ := New(common.Hash{}, NewDatabase(db))
```

Comment:
I think you can merge the two lines: `statedb, _ := New(common.Hash{}, NewDatabase(ethdb.NewMemDatabase()))`